docs/source/user_guide/public_user_instructions.rst

```diff
@@ -0,0 +1,46 @@
+**************************
+Public User Instructions
+**************************
+
+If you do not have an account, this section contains all of the information regarding account creation and using the app.
+If you do have an account, see the "Instructions" section of the manual.
+
+Home View
+==============================
+
+This is the home page of the Birdspotter app. Public users are allowed to see the map view of the public datasets.
+
+.. _public_home_view:
+.. figure:: static/home_view.png
```

Comment:
Can we use an image of the home screen that an actual public user would see. Currently it is showing the view as an admin user.